lib/modules/subredditInfo.js

```diff
@@ -61,7 +61,8 @@ modules['subredditInfo'] = {
 	srRe: /\/r\/(\w+)(?:\/(new|rising|controversial|top))?\/?$/i,
 	addListeners: function(ele) {
 		ele = ele || document.body;
-		var subredditLinks = document.body.querySelectorAll('.listing-page a.subreddit, .explore-page a.subreddit, .comment .md a[href^="/r/"], .side a[href^="/r/"], .usertext-body a[href^="/r/"], .trending-subreddits a[href^="/r/"]');
+        var querySelectors = ['.listing-page a.subreddit','.explore-page a.subreddit','.comment .md a[href^="/r/"]','.comment .md a[href*="reddit.com/r/"]','.side a[href^="/r/"]','.side a[href*="reddit.com/r/"]','.usertext-body a[href^="/r/"]','.usertext-body a[href*="reddit.com/r/"]','.trending-subreddits a[href^="/r/"]','.trending-subreddits a[href*="reddit.com/r/"]'];
```

Comment:
sorry, I should add 2 things:

1) @rageandqq thank you for submitting this PR!

2) I'm not suggesting it wasn't already bananas before you added more to that list. :-)